pilot/pkg/model/jwks_resolver.go

```diff
@@ -365,6 +372,7 @@ func (r *JwksResolver) getRemoteContentWithRetry(uri string, retry int) ([]byte,
 
 	for i := 0; i < retry; i++ {
 		body, err := getPublicKey()
+		fmt.Println(r.retryInterval)
```

Comment:
This should be deleted from the PR.

Reply:
oh yeah, I forgot to remove this. Will do it.